src/main/webapp/WEB-INF/view/admin.jsp

```diff
@@ -8,10 +8,18 @@
   <link rel="stylesheet" href="/css/main.css">
 </head>
 <body>
+  <!-- This is redirecting the user if they are not an admin -->
+  <%  if (request.getSession().getAttribute("user") == null) {
+        String redirectURL = "http://localhost:8080/";
+        response.sendRedirect(redirectURL);
+      } else if (!(boolean)request.getSession().getAttribute("adminStatus")) {
```

Comment:
Since the same code is executed in both cases you could condense to one if statement with an OR

`if  (request.getSession().getAttribute("user") == null || !(boolean)request.getSession().getAttribute("adminStatus")) {
  String redirectURL = "http://localhost:8080/";
  response.sendRedirect(redirectURL);
}`

Reply:
Condense

Reply:
Condense